src/jit/codegenxarch.cpp

```diff
@@ -1280,42 +1280,31 @@ void CodeGen::genCodeForDivMod(GenTreeOp* treeNode)
             gcInfo.gcMarkRegSetNpt(RBM_RDX);
         }
 
-        if (divisor->isContainedIntOrIImmed())
-        {
-            GenTreeIntConCommon* divImm = divisor->AsIntConCommon();
-            assert(divImm->IsIntCnsFitsInI32());
-            ssize_t imm = divImm->IconValue();
-            assert(isPow2(abs(imm)));
-            genCodeForPow2Div(treeNode->AsOp());
-        }
+        // Perform the 'targetType' (64-bit or 32-bit) divide instruction
+        instruction ins;
+        if (oper == GT_UMOD || oper == GT_UDIV)
+            ins = INS_div;
         else
-        {
-            // Perform the 'targetType' (64-bit or 32-bit) divide instruction
-            instruction ins;
-            if (oper == GT_UMOD || oper == GT_UDIV)
-                ins = INS_div;
-            else
-                ins = INS_idiv;
+            ins = INS_idiv;
             
-            emit->emitInsBinary(ins, size, treeNode, divisor);
+        emit->emitInsBinary(ins, size, treeNode, divisor);
             
-            // Signed divide RDX:RAX by r/m64, with result
-            //    stored in RAX := Quotient, RDX := Remainder.
-            // Move the result to the desired register, if necessary
-            if (oper == GT_DIV || oper == GT_UDIV)
+        // Signed divide RDX:RAX by r/m64, with result
```

Comment:
> "ignore whitespace changes"

If we add `w=1` to the URL query, then whitespace is ignored:

* diff view: https://github.com/dotnet/coreclr/pull/5871/files?w=1
* commit view: https://github.com/dotnet/coreclr/pull/5871/commits/651c419d273a537020eb3442ad5be84c60bb84cd?w=1

-- see more neat/cheat tricks like `ts` for tab space etc. here:  http://git.io/sheet